src/components/Common/Layout/TaskColumn.tsx

```diff
@@ -0,0 +1,65 @@
+import { useQuery } from '@apollo/client';
+import { TaskCard } from '@components/Card';
+import { GET_TASKS } from '@gql/queries.graph';
+import { Task } from '@models/task.model';
+import classNames from 'classnames';
+import React, {
+    FC, PropsWithChildren, Suspense, useContext,
+} from 'react';
+import { formatTag as formatSpecialChars } from '@utils/tags';
+import { Status, Tag } from '@models/common.model';
+import { TaskContext } from '@ctx/task.ctx';
+import Loader from '../Loader';
+
+interface ColumnProps extends PropsWithChildren {
+    className: string;
+    status: Status;
+}
+
+const TaskColumn: FC<ColumnProps> = ({ status }) => {
+    const { tasks, setTasks } = useContext(TaskContext)!;
+
```

Comment:
With suspense, there's some behavior lacking in apollo client (at least lacking in my knowledge and google searchs 🥲